tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Services/ContactDetailServiceTests.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
```

Comment:
Are the following `using` statements required?

```c#
using System.Threading.Tasks;
using AutoFixture.NUnit3;
```

Reply:
Those are unused. Removed those